applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_integration_schemes.cpp

```diff
@@ -0,0 +1,130 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Anne van de Graaf
+//
+
+#include "custom_elements/lobatto_integration_scheme.h"
+#include "includes/expect.h"
+#include "tests/cpp_tests/geo_mechanics_fast_suite.h"
+
+using namespace Kratos;
+
+namespace
+{
+
+std::unique_ptr<IntegrationScheme> MakeLobattoIntegrationScheme(std::size_t NumberOfPoints)
+{
+    return std::make_unique<LobattoIntegrationScheme>(NumberOfPoints);
+}
+
+double SumOfWeights(const Geo::IntegrationPointVectorType& rIntegrationPoints)
+{
+    auto weights = std::vector<double>{};
+    std::transform(rIntegrationPoints.begin(), rIntegrationPoints.end(), std::back_inserter(weights),
+                   [](const auto& rIntegrationPoint) { return rIntegrationPoint.Weight(); });
+    return std::accumulate(weights.cbegin(), weights.cend(), 0.0);
+}
+
+void ExpectLocalCoordinatesIncludeRangeBounds(const Geo::IntegrationPointVectorType& rIntegrationPoints)
+{
+    constexpr auto tolerance = 1.0e-6;
+
+    auto contains_lower_bound_of_xi = [tolerance](const auto& rPoint) {
```

Comment:
I'd suggest the following name, since it's not really about 'contains' but more about checking whether a specific point is the lower bound
```suggestion
    auto is_lower_bound_of_xi = [tolerance](const auto& rPoint) {
```

Reply:
Good suggestion. I have renamed the lambda expression to `is_at_lower_bound_of_xi`.